src/main/java/nl/_42/jarb/populate/SqlDatabasePopulator.java

```diff
@@ -67,7 +67,13 @@ public void execute() {
             populator.addScript(resource);
             
             JdbcUtils.doWithConnection(dataSource, connection -> {
-                populator.populate(connection);
+                boolean autoCommit  = connection.getAutoCommit();
+                try {
+                    connection.setAutoCommit(true);
+                    populator.populate(connection);
```

Comment:
Or perform a `connection.commit()` after `populator.populate(connection);` when autoCommit is false

Reply:
Seems cleaner as the stateful connection settings are not changed temporarly

Reply:
Logic could even be moved to `JdbcUtils.doWithConnection(dataSource, (connection) -> {}, commit = false)`

When commit == true and connection autoCommit is false, commit after callback